### PR TITLE
fixed alignment of login btn and logo

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link: https://trello.com/b/FTaaANh4/be-prepared-application

### CODE REVIEW

File: src/Components/Header.js [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-            
-        )
-                
-    }
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.onClickLoginBtn = this.onClickLoginBtn.bind(this);
+  }
+  onClickLoginBtn(e) {
```
Comment (abdulhaqshahg) [inline src/Components/Header.js]: This code should not be part of this task.

File: src/Components/Header.js [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
+      <nav id="navigation" className="navigation-nav-container">
+        <a className="navigation-logo" href="/Home">
+          Be Prepared
+        </a>
 
+        <div className="navigation-auth">
+          <button
+            className="navigation-auth-login-button"
+            onClick={this.onClickLoginBtn}
+          >
```
Comment (abdulhaqshahg) [inline src/Components/Header.js]: should properly indent 

`<button className --------------->
     loging
</button>`

File: src/Components/Header.js [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
+    this.onClickLoginBtn = this.onClickLoginBtn.bind(this);
+  }
+  onClickLoginBtn(e) {
+    this.props.history.push("/login");
+    e.preventDefault();
+  }
+  render() {
+    return (
+      <nav id="navigation" className="navigation-nav-container">
+        <a className="navigation-logo" href="/Home">
```
Comment (abdulhaqshahg) [inline src/Components/Header.js]: href is that working??

File: src/Components/Header.js [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
-            </nav>
-            
-            
-        )
-                
-    }
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.onClickLoginBtn = this.onClickLoginBtn.bind(this);
```
Comment (abdulhaqshahg) [inline src/Components/Header.js]: same as below mentioned

File: src/Components/Header.js [hunk 5]
```diff
@@ -1,24 +1,32 @@
-import React, {Component} from 'react';
+import React, { Component } from "react";
```
Comment (abdulhaqshahg) [inline src/Components/Header.js]: "react" should be in 'react' as we are taking this in all files like this so the pattern should be remain same. or we have to change every where

File: src/App.css [hunk 6]
```diff
@@ -42,59 +42,69 @@
...
 }
 .navigation-logo {
-  margin: 0 10px;
+  margin: 10px 10px;
   height: 100%;
-  display: flex;
   align-items: center;
-  transition: opacity,.2s;
+  transition: opacity, 0.2s;
   /* background-image: url(https://static.codingame.com/assets/spritesheets/navigation.6e3a4d9e.png); */
```
Comment (abdulhaqshahg) [inline src/App.css]: remove commented line

File: src/App.css [hunk 7]
```diff
@@ -42,59 +42,69 @@
...
   align-items: center;
   height: 100%;
   padding: 0 20px;
 }
-#navigation .navigation_nav-container {
+#navigation .navigation-nav-container {
   display: flex;
   justify-content: center;
-  align-items: center;
+  /* align-items: center; */
```
Comment (abdulhaqshahg) [inline src/App.css]: remove commented line

**Review by abdulhaqshahg — CHANGES_REQUESTED** [review]

File: src/Components/Header.js (hunk 3 above)
Comment (Humnaa) [reply src/Components/Header.js]: in future ..it will use

File: src/Components/Header.js (hunk 1 above)
Comment (Humnaa) [reply src/Components/Header.js]: removed

File: src/Components/Header.js (hunk 3 above)
Comment (Humnaa) [reply src/Components/Header.js]: removed

File: src/Components/Header.js (hunk 4 above)
Comment (Humnaa) [reply src/Components/Header.js]: done

File: src/Components/Header.js (hunk 5 above)
Comment (Humnaa) [reply src/Components/Header.js]: done .....but when i use to press strl+shift+i for indentation.....it automatically convert it into "react"

File: src/App.css (hunk 6 above)
Comment (Humnaa) [reply src/App.css]: done

File: src/App.css (hunk 7 above)
Comment (Humnaa) [reply src/App.css]: done

File: src/Components/Header.js (hunk 2 above)
Comment (Humnaa) [reply src/Components/Header.js]: sorry .....couldn't get ur words

Comment (abdulhaqshah) [reply src/Components/Header.js]: `<button className="navigation-auth-login-button"onClick={this.onClickLoginBtn} > </button>`

**Comment by abdulhaqshah** [comment]
image


@Humnaa Login Button should be the same size of Signup Button. It would be great if you make a logo text a bit high. Its not looking prettier.

**Comment by abdulhaqshah** [comment]
image

Why the color of the LOGO is black, it should be white

File: src/App.css [hunk 8]
```diff
@@ -42,59 +37,67 @@
...
+  width:170px
 }
-.navigation-auth .navigation-auth_login-button {
+.navigation-auth .navigation-auth-login-button {
   text-align: center;
-  background-color: hsla(0,0%,100%,.08);
+  background-color: hsla(0, 0%, 100%, 0.08);
   color: #fff;
-  font-size: 14px;
+  font-size: 16px;
```
Comment (abdulhaqshah) [inline src/App.css]: font-size make it 14 like the singup button

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 8 above)
Comment (Humnaa) [reply src/App.css]: I've changed its font size to 14px

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**